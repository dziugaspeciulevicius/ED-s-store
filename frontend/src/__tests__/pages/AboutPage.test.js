import React from "react";
import { shallow } from "enzyme";
import "../../setupTests";
import AboutPage from "../../pages/AboutPage.component";

describe("<AboutPage />", () => {
  it("Renders <AboutPage /> component correctly", () => {
    // shallow creates instance of a component
    expect(shallow(<AboutPage />)).toMatchSnapshot();
  });
});
