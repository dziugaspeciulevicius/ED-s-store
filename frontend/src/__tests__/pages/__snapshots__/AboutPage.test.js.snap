// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutPage /> Renders <AboutPage /> component correctly 1`] = `ShallowWrapper {}`;
